### DISCUSSION TITLE
Recreate TDP Apache repos as GitHub forks

### DISCUSSION THREAD

**Discussion by Nuttymoon** [TOSIT-IO/TDP#37] [Ideas]
It would be great to recreate all the TDP Apache repos (Hadoop, Hive, HBase, etc.) through GitHub forks functionality. This will:
- Make it clear that the repos are real forks from the Apache softwares
- Ease the maintenance process (e.g. updating branch from the source repo)

Example with Kafka: https://github.com/TOSIT-IO/kafka
![image](https://user-images.githubusercontent.com/11584800/175101176-8dad3f1d-45bd-4a93-b67b-38cc76cdf876.png)

**Comment by Pierrotws** [comment]
done
